### PR TITLE
Added product security label

### PR DESCRIPTION
**Pull request by brian-avery**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 1]
```diff
@@ -21,6 +21,7 @@ about: Suggest an idea to improve Istio
 [ ] Test and Release
 [ ] User Experience
 [ ] Developer Infrastructure
+[ ] Product Security
```
Comment (howardjohn) [inline .github/ISSUE_TEMPLATE/feature_request.md:24]: I don't think a user will know whether to select "security" or "product security" without more context. I also doubt they understand "Configuration Infrastructure" but that is unrelated

Comment (brian-avery) [reply .github/ISSUE_TEMPLATE/feature_request.md:24]: Agree, that's a challenge that I think we have and I'm not quite sure how to get around it yet. We have a security group in which security issues shouldn't be talked about (only security related features) and a product security group that sounds similar to the security group. We don't have a clear way of differentiating the two.

Comment (rshriram) [reply .github/ISSUE_TEMPLATE/feature_request.md:24]: Call it as vulnerability report?

Comment (brian-avery) [reply .github/ISSUE_TEMPLATE/feature_request.md:24]: Vulnerabilities shouldn't be reported through GitHub issues. They should be reported through the mailing list. This is for issues found on GitHub that need PSC attention or functionality being added. For instance, functionality was just added to scan base images for vulnerabilities. There are also issues that have been found by other groups on GitHub that warranted PSC attention. 

I think for now, we can close this as the audience who would be using this label is likely to be applied by people who know that it exists already. That said, my argument still stands about confusion between security features in Istio and security features for Istio.
